lib/utils.js

```diff
@@ -26,6 +27,20 @@ function parseRoleId(role) {
   return matches && matches[1] || -1
 }
 
+/**
+ * Determine if string is a valid unicode emoji
+ *
+ * @param {string} emoji unicode emoji
+ * @returns {boolean}
+ */
+function validEmoji(emoji) {
+  try {
+    return emoji.match(emojiRegex)[0] !== null
```

Comment:
How DARE you use such a horrible approach.

Use `emojiRegex.test(emoji)` instead. No need for try/catch here either.

Reply:
Or just remove this function entirely and directly use `.test()` where it's necessary

Reply:
Oh snap! I can't believe I DAREd due such a horrible thing.

It shall be fixed at once good sir.